metadata_service/proxy/atlas_proxy.py

```diff
@@ -596,8 +599,42 @@ def get_table_by_user_relation(self, *, user_email: str, relation_type: UserReso
 
         return {'table': results}
 
-    def get_frequently_used_tables(self, *, user_email: str) -> Dict[str, Any]:
-        pass
+    def get_frequently_used_tables(self, *, user_email: str) -> Dict[str, List[PopularTable]]:
+        user = self._driver.entity_unique_attribute(self.USER_TYPE, qualifiedName=user_email).entity
+
+        readers_guids = []
+        for r in user['relationshipAttributes'].get('entityReads'):
+            entity_status = r['entityStatus']
+            relationship_status = r['relationshipStatus']
+
+            if entity_status == 'ACTIVE' and relationship_status == 'ACTIVE':
+                readers_guids.append(r['guid'])
+
+        readers = self._extract_entities(self._driver.entity_bulk(guid=readers_guids, ignoreRelationships=True))
+
+        _results = {}
+        for reader in readers:
+            entity_uri = reader.attributes.get(self.ENTITY_URI_KEY)
+            count = reader.attributes.get('count')
+
+            if count > 0:
+                details = self._extract_info_from_uri(table_uri=entity_uri)
+
+                _results[count] = dict(cluster=details.get('cluster'),
+                                       name=details.get('name'),
+                                       schema=details.get('db'),
+                                       database=details.get('entity'))
+
+        sorted_counts = list(_results.keys())
```

Comment:
you mean `sorted(_results.keys())` to sort it. 
You can also use OrderedDict.
```
import collections
for key, value in collections.OrderedDict(sorted(_results.items())):
```

Reply:
agree, should be sorted. I am not sure though if OrderedDict is of any use here - it should be used to `remember the order entries were added` and that's not of any use here. we need to sort the keys by value not order of inserting